### PR TITLE
fix duplicate column name failing xmatch

### PR DESCRIPTION
**Pull request by lijuan**

### CODE REVIEW

File: dl/helpers/utils.py [hunk 1]
```diff
@@ -106,6 +106,22 @@ def convert(inp,outfmt='pandas',verbose=False,**kwargs):
        df = convert(inp,'pandas',na_values='Infinity') # na_values is a kwarg; adds 'Infinity' to list of values converter to np.inf
 
     """
+    index = inp.find('\n')
```
Comment (acscott) [inline dl/helpers/utils.py:109]: This is good Wendy.  I might had a comment as to what we are doing in this "paragraph" of code and why.

**Review by acscott — APPROVED** [review]
Excellent fix!

File: dl/helpers/utils.py (hunk 1 above)
Comment (lijuan) [reply dl/helpers/utils.py:111]: Comments are added. Thanks for checking!

**Comment by rnikutta** [comment]
Hi @lijuan Sorry just curious: what is this fix for? Could you please post an example 'inp' that the new fix is handling? Thank you!

**Comment by lijuan** [comment]
@rnikutta Here is the problematic query that does not work for Arjun:

    query="""
SELECT S.*, L.*
FROM mydb://andes_viresults2 as S, nsc_dr2.object as L
where q3c_join(S.target_ra, S.target_dec, L.ra, L.dec, (1/3600.0))
    """
    response = queryClient.query(sql = query,fmt='table', profile='db01')

The issue is in nsc_dr2.object table there is an 'id' column, while q3c_join would also generate an 'id' column, so if the user try to convert the query result to 'table' or 'votable', it would fail. My fix is to add an identifier to the duplicated column names.

**Comment by mjfitzpatrick** [comment]
q3c_join() doesn't 'generate' any columns,  For example, try in the
database:

select * from usno.a2 where q3c_join(0.0, 0.0, raj2000, dej2000, 1.0) limit
2;

with and without the q3c.  There's already an 'id' column in usno.a2.

I may be misunderstanding, your query appears to be truncating the "s.,L."
part of the query..

**Comment by lijuan** [comment]
Sorry, my mistake. The 'id' column is not from q3c_join, instead, it is from the user's mydb table, so if there are the same column name appears in both tables, the xmatch result cannot be converted to 'table' or 'votable' format. That is why Arjun's xmatch query failed at the beginning.

**Comment by mjfitzpatrick** [comment]
I realized what you meant after sending the message.  So, to be clear, it
is still possible for a user query to create duplicate column names (e.g.
in CSV output), this only addresses an internal conversion using the 'fmt'
parameter).

**Comment by lijuan** [comment]
Yes, if a user query create duplicate column names in csv output, it is ok. It only incurs problems when they want to get the 'table' fmt or 'votable' fmt.

**Pull request merged**